src/sentry/eventtypes/base.py

```diff
@@ -36,3 +36,6 @@ def get_metadata(self):
         return {
             'title': title,
         }
+
+    def to_string(self, data):
```

Comment:
Its not yet used, but the goal to utilize this anytime we need a description of an event.